LandSandBoat/server
# [LUA] Tuchulcha ambush fixes

Stealth trait plus the fact that ant_positions was 1.8 yalms below ground made him reliably not aggro as ninja or ranger



**_I affirm:_**

## What does this pull request do?

Sheep in Antlion's Clothes has 4 mobs, one of which is the main NM. They start underground with the ambush aggro behavior. This has a pretty tight window on distance to trigger the `pit_ambush` mobskill, and using ninja/ranger with stealth trait makes that window even smaller. Turns out all the ant locations in `IDs.lua` were reliably 1.8 yalms below ground (yes i checked them all). Furthermore, Tuchulcha's spawn position was also a bit below ground.

This PR does two things:
- adds another valid ant location to all 3 instances from Tuchulcha's spawn positions, respectively. It now sets Tuchulcha to a random location on instance init
- 2nd thing this does is ensures all positions in each instance are at ground level so a player running by, even with stealth trait, will trigger the ambush

## Steps to test these changes

`!zone boneyard gully`
`!addkeyitem MIASMA_FILTER`
start instance and run around, finding the hidden ants. Note that Tuchulcha goes under every 25% hp, so one instance one can test most positions.
